analysis/src/python/data_collection/hyperskill/api/steps.py

```diff
@@ -0,0 +1,83 @@
+from dataclasses import dataclass, field
```

Comment:
It would be super useful in the future if you describe in docs or somewhere all fields for hyperskill objects that you use